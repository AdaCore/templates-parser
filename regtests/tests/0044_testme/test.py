from test_support import *

run('testme', ["testme40.tmplt"])
