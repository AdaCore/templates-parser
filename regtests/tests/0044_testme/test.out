

include2.tmplt

1> 

2> two

3> 

4> 

5> 

ok

